File: 2486-append-characters-to-string-to-make-subsequence/2486-append-characters-to-string-to-make-subsequence.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        j = 0;
        for i in range(len(s)):
            if s[i] == t[j]: j += 1;
            if j == len(t): return 0;
        return len(t) - j;

File: 2486-append-characters-to-string-to-make-subsequence/2486-append-characters-to-string-to-make-subsequence.java
class Solution {
    public int appendCharacters(String s, String t) {
        int j = 0;
        for(int i = 0; i < s.length(); ++i) {
            if(s.charAt(i) == t.charAt(j)) ++j;
            if(j == t.length()) return 0;
        }
        return t.length() - j;
    }
}